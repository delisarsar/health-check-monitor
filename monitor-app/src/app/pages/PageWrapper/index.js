import React from "react";
import PropTypes from "prop-types";
import Header from "../../components/Header";
import { Box } from "@material-ui/core";

const PageWrapper = (props) => {
  const { children } = props;
  return (
    <>
      <Header />
      <Box pt={3} pl={3}>
        {children}
      </Box>
    </>
  );
};

export default PageWrapper;

PageWrapper.propTypes = {
  children: PropTypes.node,
};
