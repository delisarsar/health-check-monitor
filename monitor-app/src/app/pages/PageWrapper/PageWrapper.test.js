import React from "react";
import { render } from "@testing-library/react";
import PageWrapper from ".";
import { Typography } from "@material-ui/core";

describe("<PageWrapper />", () => {
  test("renders correctly", () => {
    const { container } = render(
      <PageWrapper>
        <Typography>learn react</Typography>
      </PageWrapper>
    );
    expect(container).toMatchSnapshot();
  });
});
