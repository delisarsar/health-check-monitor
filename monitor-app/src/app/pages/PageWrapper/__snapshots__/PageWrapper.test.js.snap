// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    class="makeStyles-root-1"
  >
    <header
      class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary makeStyles-appBar-2 MuiPaper-elevation4"
    >
      <div
        class="MuiBox-root MuiBox-root-4"
      >
        <img
          alt="Logo"
          height="49"
          src="logo.png"
          width="108"
        />
        ;
      </div>
    </header>
  </div>
  <div
    class="MuiBox-root MuiBox-root-5"
  >
    <p
      class="MuiTypography-root MuiTypography-body1"
    >
      learn react
    </p>
  </div>
</div>
`;
